docs/reference/experiment-config.txt

```diff
@@ -66,17 +66,18 @@ Top-Level Fields
 ``min_checkpoint_period``
   Instructs Determined to take periodic checkpoints of each trial during
   training. If set, this variable specifies the maximum length that a trial
-  can be trained without a checkpoint, in terms of records, batches or epochs
+  can be trained without a checkpoint, in terms of records, batches, or epochs
   (see :ref:`Training Units<experiment-configuration_training_units>`);
   if this limit is reached, a checkpoint of the trial is
   taken. There are three other situations in which a trial might be
-  checkpointed: (a) during training, a model may be checkpointed to
-  allow the trial's execution to be suspended and later resumed on
-  a different Determined agent (b) when the trial's experiment is
-  completed, to allow the resulting model to be exported from
-  Determined (e.g., for deployment) (c) before the search method makes
-  a decision based on a validation of a trial, to maintain consistency
-  in the event of a failure.
+  checkpointed:
+
+  1. During training, a model may be checkpointed to allow the trial's execution
+     to be suspended and later resumed on a different Determined agent
+  2. When the trial's experiment is completed, to allow the resulting model to
+     be exported from Determined (e.g., for deployment)
+  3. Before the search method makes a decision based on a validation of a trial,
+     to maintain consistency in the event of a failure.
```

Comment:
more of a question than a change request or anything, should this have a period at the end or not?

Reply:
Good catch, I think periods would be good -- or at any rate being consistent.